Clarify that credentials are not required (#169)
Caltech-IPAC/irsa-tutorials: tutorials/parquet-catalog-demos/wise-allwise-catalog-demo.md

+++

This AllWISE catalog is stored in an [AWS S3](https://aws.amazon.com/s3/) cloud storage bucket.
To connect to an S3 bucket we just need to point the reader at S3 instead of the local filesystem.
(Here, a "reader" is a python library that reads parquet files.)

> We'll use [pyarrow.fs.S3FileSystem](https://arrow.apache.org/docs/python/generated/pyarrow.fs.S3FileSystem.html) for this because it is recognized by every reader in examples below, and we're already using pyarrow.

Author: Because it's covered in the cloud access notebook that I added a link to and I felt that it interrupted the flow of this paragraph much more than it added. Do you think it's particularly valuable here?

> To access without credentials, we'll use the keyword argument `anonymous=True`.

Reviewer: ```suggestion
To avoid an error about credentials, we'll use the keyword argument `anonymous=True`.
```
